apps/web/src/components/Shared/GlobalModals.tsx

```diff
@@ -99,6 +102,14 @@ const GlobalModals: FC = () => {
       >
         <Invites />
       </Modal>
+      <Modal
+        size="lg"
+        title={t`Select a NFT`}
+        show={showNftAvatarModal}
+        onClose={() => setShowNftAvatarModal(false)}
+      >
+        <NftAvatarModal />
+      </Modal>
```

Comment:
Why we need this in global modal? local will be fine for this, cuz we are not using across screens